### PR TITLE
KEYCLOAK-13127 Update Account Console to Account REST API v1

### PR DESCRIPTION
**Pull request by vmuzikar**
Base tests and new account console tests: https://keycloak-jenkins.com/job/universal-test-pipeline-server/1989/
The test failures are unrelated to this change.

### CODE REVIEW

File: common/src/main/java/org/keycloak/common/enums/AccountRestApiVersion.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+import java.util.concurrent.ConcurrentHashMap;
+
+/**
+ * @author Vaclav Muzikar <vmuzikar@redhat.com>
+ */
+public enum AccountRestApiVersion {
+    V1("v1");
+
+    public static final AccountRestApiVersion DEFAULT = V1;
+    private static final Map<String,AccountRestApiVersion> ENUM_MAP;
```
Comment (slaskawi) [inline common/src/main/java/org/keycloak/common/enums/AccountRestApiVersion.java:31]: You may use an [EnumMap](https://docs.oracle.com/javase/7/docs/api/java/util/EnumMap.html).

File: common/src/main/java/org/keycloak/common/enums/AccountRestApiVersion.java [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+    static {
+        Map<String, AccountRestApiVersion> map = new ConcurrentHashMap<>();
```
Comment (slaskawi) [inline common/src/main/java/org/keycloak/common/enums/AccountRestApiVersion.java:34]: Doesn't need to be concurrent - Enums are initialized only once.

**Review by slaskawi — DISMISSED** [review]
LGTM, only some small nitpicks found. I'm leaving the decision whether to implement them (or not) to you. 

BTW, do we plan to support some kind of aliases? Something like `latest`, which would always redirect you to the latest version? I think it would be very convenient (especially if we plan to be backwards compatible to some extend).

File: common/src/main/java/org/keycloak/common/enums/AccountRestApiVersion.java (hunk 1 above)
Comment (vmuzikar) [reply common/src/main/java/org/keycloak/common/enums/AccountRestApiVersion.java:31]: I don't think we want a Map where Enum is the key here.

File: common/src/main/java/org/keycloak/common/enums/AccountRestApiVersion.java (hunk 2 above)
Comment (vmuzikar) [reply common/src/main/java/org/keycloak/common/enums/AccountRestApiVersion.java:34]: Good catch, I'll fix that. Thanks.

**Review by abstractj — APPROVED** [review]

**Pull request merged**